developers/wallets/native/index.md

```diff
@@ -119,23 +124,70 @@ The `request` value is URL encoded. Its contents look like this when unencoded:
 
 There are currently two possible protocols that Issuers may use with CHAPI in order to issue Verifiable Credentials to be stored in the user's digital wallet: `vcapi` and `OID4VCI`.
 
-Depending on what you declared in your `acceptedProtocols` you may receive either or both as properties in the `protocols` object carried in the `request` query parameter. It is up to wallet providers to decide (or they may delegate this to advanced users) which protocol to use if multiple protocols are supported.
+Depending on what you declared in your `acceptedProtocols` you may receive either or both as properties in the `protocols` object carried in the `request` query parameter. Your Wallet's code (i.e. in `/switchboard`) should decide which protocol to use (or it may delegate this choice to advanced users) when multiple protocols are available.
```

Comment:
```suggestion
Depending on what you declared in your `acceptedProtocols` you may receive either or both as properties in the `protocols` object carried in the `request` query parameter. Your wallet's code (i.e. in `/switchboard`) should decide which protocol to use (or it may delegate this choice to advanced users) when multiple protocols are available.
```